src/Drivers/OpenAI/Client.php

```diff
@@ -15,10 +15,12 @@ class Client
     public function __construct(
         public readonly string $apiKey,
         public readonly string $url,
+        public readonly ?string $organization = null,
     ) {
-        $this->client = Http::withHeaders([
-            'Authorization' => sprintf('Bearer %s', $this->apiKey),
-        ])->baseUrl($this->url);
+        $this->client = Http::withHeaders(array_filter([
+            'Authorization' => $this->apiKey !== '' && $this->apiKey !== '0' ? sprintf('Bearer %s', $this->apiKey) : null,
```

Comment:
@sixlive I think the formatting got a bit heavy handed here (`composer format did this to mine, too`) but I don't think passing a `API` key is valuable since that will surely `401`. Instead I think it should Raise, but also type-checking won't let a `null` be passed to the API.

Reply:
Probably out of scope for this PR, but if the API key is optional then we likely should have it be set as optional as well.